servicetalk-http-api/src/main/java/io/servicetalk/http/api/DefaultHttpCookiePair.java

```diff
@@ -87,7 +87,7 @@ public static HttpCookiePair parseCookiePair(final CharSequence sequence, int na
 
     static HttpCookiePair parseCookiePair0(final CharSequence sequence, int nameStart, int nameLength, int valueEnd) {
```

Comment:
While you are here, can you please make this method private?

Btw, are the previous two `parseCookiePair` method have to be public? They currently used only by `HeaderUtils`, where `HeaderUtils#parseCookiePair` is also public. If any of these methods have to be public, can we consolidate them all in one place?

Reply:
> While you are here, can you please make this method private?

done.

I can make both of the `parseCookiePair` methods package private, they may have been used by the `netty` packages at some point during development. i'll do this as a followup PR.

Reply:
package private PR: https://github.com/apple/servicetalk/pull/1296